### DISCUSSION TITLE
Allow settings custom container volumes instead of the default "nextcloud_aio_xxx"

### DISCUSSION THREAD

**Discussion by Joly0** [nextcloud/all-in-one#666] [Ideas] [answered]
It would be great if one could set a master folder and define that as an environment variable for all containers which are spawned. These containers then would map their volumes to the master folder instead of the docker folder.

**Comment by szaimen** [comment, marked as answer]
Hello, this goes a bit into the same direction like https://github.com/nextcloud/all-in-one/discussions/359
